src/main/kotlin/no/nav/familie/ba/sak/kjerne/behandlingsresultat/YtelsePersonUtils.kt

```diff
@@ -106,56 +107,71 @@ object YtelsePersonUtils {
             }
         )
 
-    private fun erEndring(
+    // metoden utleder YtelsesPersonResultat om det finnes en endring ellers sender null
+    private fun utledYtelsePersonResultatVedEndring(
         behandlingsresultatPerson: BehandlingsresultatPerson,
         segmenterLagtTil: LocalDateTimeline<BehandlingsresultatAndelTilkjentYtelse>,
         segmenterFjernet: LocalDateTimeline<BehandlingsresultatAndelTilkjentYtelse>,
         ytelsePerson: YtelsePerson,
         inneværendeMåned: YearMonth
-    ): Boolean {
+    ): YtelsePersonResultat? {
         val nesteMåned = inneværendeMåned.nesteMåned()
-        val stønadSlutt =
-            behandlingsresultatPerson.andeler.maxByOrNull { it.stønadFom }?.stønadTom
-                ?: TIDENES_MORGEN.toYearMonth()
+        val stønadSlutt = behandlingsresultatPerson.andeler.maxByOrNull { it.stønadFom }?.stønadTom
+            ?: TIDENES_MORGEN.toYearMonth()
 
-        val forrigeStønadSlutt =
-            behandlingsresultatPerson
-                .forrigeAndeler
-                .maxByOrNull { it.stønadFom }
-                ?.stønadTom ?: TIDENES_MORGEN.toYearMonth()
+        val forrigeStønadSlutt = behandlingsresultatPerson.forrigeAndeler
+            .maxByOrNull { it.stønadFom }?.stønadTom ?: TIDENES_MORGEN.toYearMonth()
 
         val opphører = stønadSlutt.isBefore(nesteMåned)
 
-        if (behandlingsresultatPerson.søktForPerson) {
+        return if (behandlingsresultatPerson.søktForPerson) {
             val beløpRedusert = (segmenterLagtTil + segmenterFjernet).isEmpty() &&
-                (behandlingsresultatPerson.forrigeAndeler.sumOf { it.sumForPeriode() } - behandlingsresultatPerson.andeler.sumOf { it.sumForPeriode() }) > 0
+                (
+                behandlingsresultatPerson.forrigeAndeler.sumOf { it.sumForPeriode() } -
+                    behandlingsresultatPerson.andeler.sumOf { it.sumForPeriode() }
+                ) > 0
```

Comment:
Er ikke dette det samme som
```suggestion
                behandlingsresultatPerson.forrigeAndeler.sumOf { it.sumForPeriode() } >
                    behandlingsresultatPerson.andeler.sumOf { it.sumForPeriode() }
```
?

Reply:
Ja, det er samme. Men det er eksisterende kode, får vi noen forbedring ved å endre dette?